doc/classes/PopupMenu.xml

```diff
@@ -463,6 +492,7 @@
 			<argument index="1" name="disabled" type="bool">
 			</argument>
 			<description>
+				Determines whether the item shortcut of the specified [code]idx[/code] will be disabled or not.
```

Comment:
This is a setter function, so it modifies a property directly. In this case `Disables the shortcut item at index [code]idx[/code].`
Or if you want to be more precise, you could say `Disables or enables`.